scripts/functional-tests.sh

```diff
@@ -582,6 +582,24 @@ test_ilab_chat_server_logs(){
     fi
 }
 
+test_ilab_help_without_config() {
+    result=$(ilab --config=test-test.yaml model evaluate --help)
```

Comment:
@reidliu41 Thanks for adding a second case. Could we please add a case where the command fails because it doesn't have the config? That way we can make sure your test is working as expected